app/konnect/org-management/audit-logging/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Audit Logging in Konnect
+badge: enterprise
+content_type: explanation
+beta: true
+---
+
+Audit logging enables administrators to better spot security risks and maintain compliance of their core infrastructure. 
+
+No system is completely secure. Audit logs provide a mechanism to catch abnormal events when those events occur. 
+If potential security incidents are not caught early, perpetrators can cause issues or even downtime with a company’s infrastructure, 
+leading to serious consequences such as outages and lost revenue.
+
+By tracking {{site.konnect_short_name}} audit logs, you gain the following benefits:
+* **Security**: System events can be used to show abnormalities to be investigated, forensic information related to breaches, or provide evidence.
+* **Compliance**: Some regulations require audit logs, or audit logs are needed to show auditors who confirm whether certain certification standards are met.
+* **Debugging**: Audit logs give insights into why there is a current level of efficiency or performance. 
```

Comment:
```suggestion
* **Debugging**: Audit logs can help pinpoint the root cause of efficiency or performance issues.
```

Reply:
Went with "determine" over "pinpoint" to avoid colloquialisms.